src/sys/signal.rs

```diff
@@ -370,6 +372,15 @@ impl SigSet {
         try!(pthread_sigmask(how, Some(self), Some(&mut oldmask)));
         Ok(oldmask)
     }
+
+    /// Suspends execution of the calling thread until one of the signals in the
+    /// signal mask becomes pending, and returns the accepted signal.
+    pub fn wait(&self) -> Result<SigNum> {
+        let mut signum: SigNum = unsafe { mem::uninitialized() };
```

Comment:
It makes sense. I propose adding a note to CONVENTIONS stating that we want to use `std::mem::uninitialized` in such cases.